src/ui/svg/icons/PlacePointer.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import Svg, { Path, G } from 'react-native-svg'
+
+import { ColorsEnum } from 'ui/theme'
+
+import { IconInterface } from './types'
+
+export const PlacePointer: React.FunctionComponent<IconInterface> = ({
```

Comment:
Ce serait top de l'ajouter à la page `AppComponents` (qui liste notamment toutes nos icones, c'est sympa pour voir si l'icone qu'on cherche est déjà in-app ou non)